ui/src/containers/Award/component/AwardItemStyle.scss

```diff
@@ -0,0 +1,35 @@
+@import '../mockVar.scss';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 30px 0px;
+}
+
+.content {
+  text-align: center;
+  padding: 10px 0px;
+  margin: 8px 0px;
+  width: 400px;
+}
+
+.prize {
+  @extend .content;
+  color: white;
+  background-color: $orange;
+  border-radius: 10px;
+  font-size: 2em;
+}
+
+.winner {
+  @extend .content;
+  color: black;
+  background-color: white;
+  font-size: 1.5em;
+}
+
+.winner--none {
+  @extend .winner;
+  border: 2px dotted black;
+}
```

Comment:
cursor pointer is required for winner-none

Reply:
fixed in d015ca467895895318a8f85c0aa5de9a02e182bc